configure

```diff
@@ -436,8 +438,23 @@ case $CAMLVERSION in
 	    echo "          Configuration script failed!"
 	    exit 1
 	fi;;
+    3.10*|3.11*|3.12*)
+	CAMLP4COMPAT="-loc loc" 
+	echo "You have Objective-Caml $CAMLVERSION. Good!";;
+    4.02.0)
+	CAMLP4COMPAT="-loc loc"
+	echo "Warning: OCaml 4.02.0 suffers from a bug inducing"
+	echo "very slow compilation times";;
```

Comment:
Very slow has no meaning. Does it take 10 hours on a modern laptop to compile it, 100, or 10000?